wc-smooth-generator.php

```diff
@@ -13,7 +13,7 @@
 defined( 'ABSPATH' ) || exit;
 
 // autoloader.
-require 'vendor/autoload.php';
+require __DIR__ . '/vendor/autoload.php';
```

Comment:
For my use case I actually want this to be `require WP_CONTENT_DIR . '/vendor/autoload.php';` as I'm using it in my custom code to generate products. Either way, I think a more robust of requiring the autoloader is needed.

Reply:
This code supposed to load `autoload.php` from `wc-smooth-generator` directory. I think you can always change whatever you need when you do custom coding. So, it shouldn't use `WP_CONTENT_DIR `. Beside, there are a lot of other plugins which use their own composer autoload.php to load packages from their own vendor directories.